executor/revoke.go

```diff
@@ -50,15 +50,6 @@ type RevokeExec struct {
 	done bool
 }
 
-// Next implements Execution Next interface.
-func (e *RevokeExec) Next(ctx context.Context) (Row, error) {
-	if e.done {
-		return nil, nil
-	}
-	e.done = true
-	return nil, errors.Trace(e.run(ctx))
-}
-
 // NextChunk implements the Executor NextChunk interface.
 func (e *RevokeExec) NextChunk(ctx context.Context, chk *chunk.Chunk) error {
```

Comment:
`e.run` can be removed and put the codes into `NextChunk`

Reply:
```go
    if e.done {
        return nil
    }
    err := e.run(ctx)
    e.done = true
    return errors.Trace(err)
```
Is it?